charts/monitoring-config/rules/mirrors.yaml

```diff
@@ -0,0 +1,14 @@
+- name: MirrorAlerts
+  rules:
+    - record: global:govuk_mirror_freshness_seconds
+      expr: |
+        time() - min(govuk_mirror_last_updated_time) by (backend)
+
+    - alert: MirrorFreshnessAlert
+      expr: |
+        govuk_mirror_freshness_seconds < 172800
+      for: 1h
```

Comment:
optional: not sure the `for` is doing much here (and it's an optional field)

Reply:
(slight doubt in the back of my mind that maybe this is working around counter resets or missing data points or something, in which case ignore me!)